sct-commons/src/test/java/org/lfenergy/compas/sct/commons/scl/dtt/LNodeTypeAdapterTest.java

```diff
@@ -14,6 +14,7 @@
 import org.lfenergy.compas.sct.commons.scl.SclRootAdapter;
 
```

Comment:
I've got 2 test failing in my IDE :
org.lfenergy.compas.sct.commons.scl.dtt.LNodeTypeAdapterTest#testHasSameContentAs
and
org.lfenergy.compas.sct.commons.scl.dtt.DataTypeTemplateAdapterTest#testImportLNodeType
org.lfenergy.compas.sct.commons.scl.dtt.DataTypeTemplateAdapterTest#testImportDTT